source/TestApp/App.xaml

```diff
@@ -25,7 +25,7 @@
                     Set this property to true to auto-size the demo LayoutAnchorable to the size of its containing control
                     when being dragged into floating mode
                 -->
-                <Setter Property="SetWindowSizeWhenOpened" Value="False" />
+                <!--<Setter Property="SetWindowSizeWhenOpened" Value="True" />-->
```

Comment:
This can be removed completely

Reply:
Thank you for the timely feedback and the hints on the documentation. I changed the documentation in a [separate commit](https://github.com/Dirkster99/AvalonDock/commit/b85b1870f0147cadcb8230966c333eeedf8a35f0) (and also adjusted the relevant Wiki sections):